apps/client/src/components/NavStyle.ts

```diff
@@ -1,7 +1,11 @@
 import { Icon, themeColor, themeSpacing } from "@datapunt/asc-ui";
 import styled, { css } from "styled-components";
 
-export const NavStyle = styled.div`
+type NavProps = {
+  noMarginBottom?: boolean;
+};
```

Comment:
After you made all those props optional in the previous step, you can replace this by importing NavProps from Nav (is has an export for some reason)

```
import { NavProps } from "./Nav";
```